config/context/spring-context.xml

```diff
@@ -106,9 +106,18 @@
     <constructor-arg name="keyPair" ref="keyPair"/>
     <constructor-arg name="irohaAPI" ref="irohaAPI"/>
   </bean>
+
+  <bean id="blockRule"
+    class="iroha.validation.rules.impl.BlockRule">
+    <constructor-arg name="irohaAPI" ref="irohaAPI"/>
+    <constructor-arg name="brvsAccountId" value="${credential.accountId}"/>
+    <constructor-arg name="keyPair" ref="keyPair"/>
+  </bean>
+
   <util:list id="rules" value-type="iroha.validation.rules.Rule">
     <ref bean="sampleRule"/>
-    <ref bean="newBrvsRule"/>
+    <ref bean="blockRule"/>
```

Comment:
Do not commit the change of rules list to the general config. Provide a test instead please